pkg/metrics/github_fetcher.go

```diff
@@ -36,6 +36,10 @@ func getAllReposForOrg(orga string) []string {
 			break
 		}
 		for _, repo := range repos_page {
+			if *repo.Disabled || *repo.Archived {
+				log.Printf("Skipping Archived or Disabled repo %s", *repo.FullName)
+				continue
+			}
```

Comment:
i'm pretty certain that the `*` prefix when using `repo` is redundant (that go will automatically pick the right receiver in this case w/o needing an explicit cast)

but this matches what's in the unmodified code, so not blocking; just wanting to inform